### ISSUE TITLE
[Feature Request] Widget layout - add background colour for day headers

### ISSUE THREAD

**Issue by val24** [plusonelabs/calendar-widget#345]
Hello,

I like the new layout of ToDo Agenda, which is easier to read. I like the fact that the line serarating day headers from events is above day header.

In the current parameter options, you can set background colour / transparancy for :
- widget header
- past events
- today's events
- future events

There is no option to set background colour for day headers, which I would find really useful.
What I would find even more useful would be an option to set a different background colour for each day of the week (or at least different between week days and weekends).

For information, I like showing days without events in order to make it easier to read for me (which, I am aware, takes space for no real information, but I find it easier to read)... and I would appreciate being able to see more easily the difference between week days and weekends.

Thanks

**Comment by yvolk** [comment]
@val24 Currently background color for Day Headers is the same as for events, i.e. the color is different for Past, Today's and Future Day Headers.

You suggest (an option) to change this behaviour to some other.

**Comment by val24** [comment]
Yes, I'd like an option to have a specific colour for day header (day
header only - not events) according to the day of the week (or only
weekdays / weekends). I'd keep a different colour for today as it works
now, though. Events, past, todays's, or future could keep the current
behaviour, which I find nice.
Thanks for your answer.

Le sam. 9 nov. 2019 à 18:37, Yuri Volkov <notifications@github.com> a
écrit :
